{"id":2261,"date":"2022-05-01T07:26:28","date_gmt":"2022-05-01T07:26:28","guid":{"rendered":"https:\/\/alynews.org\/a-refined-microbiome-fingerprint-method-tracks-sub-strain-variants-of-a-single-gut-microbe-strain-an-absence-of-sub-strain-variation-over-a-short-period-was-seen-in-sick-patients-which-may-signa\/"},"modified":"2022-05-01T07:26:28","modified_gmt":"2022-05-01T07:26:28","slug":"a-refined-microbiome-fingerprint-method-tracks-sub-strain-variants-of-a-single-gut-microbe-strain-an-absence-of-sub-strain-variation-over-a-short-period-was-seen-in-sick-patients-which-may-signa","status":"publish","type":"post","link":"http:\/\/localhost\/m\/a-refined-microbiome-fingerprint-method-tracks-sub-strain-variants-of-a-single-gut-microbe-strain-an-absence-of-sub-strain-variation-over-a-short-period-was-seen-in-sick-patients-which-may-signa\/","title":{"rendered":"A refined microbiome &#8216;fingerprint&#8217; method tracks sub-strain variants of a single gut microbe strain: An absence of sub-strain variation over a short period was seen in sick patients, which may signal impending gut dysbiosis"},"content":{"rendered":"<p><img src=\"https:\/\/www.sciencedaily.com\/images\/scidaily-icon.png\" alt=\"A refined microbiome 'fingerprint' method tracks sub-strain variants of a single gut microbe strain: An absence of sub-strain variation over a short period was seen in sick patients, which may signal impending gut dysbiosis\"><\/p>\n<p>Casey D. Morrow, Ph.D., and colleagues at the University of Alabama at Birmingham previously developed a microbiome &#8220;fingerprint&#8221; method called WSS that identifies single strains of particular gut bacteria, through analysis of metagenomics data from fecal samples. They have shown that particular strains in adults tend to remain stable over time, unless perturbed by events like antibiotics or obesity surgery. They also saw that a donor fecal transplant strain given to treat drug-resistant Clostridium difficile infections persisted in the recipient for as long as two years after the transplant.<\/p>\n<p>Morrow and Hyunmin Koo, Ph.D., refined the fingerprint method to include looking for single-nucleotide variants in KEGG metabolic pathways of a particular strain. These variants can identify sub-strains of a single strain identified by WSS. To look at sub-strains of a Bacteroides vulgatus strain, for example, Morrow and Koo examined 23 different KEGG metabolic pathways present in that bacteria.<\/p>\n<p>They have now applied this magnified analysis to monitor changes in sub-strains over shorter periods of time, days or weeks, in two key gut bacteria &#8212; B. vulgatus and Bacteroides uniformis. Comparing a small number of healthy individuals and hospitalized COVID-19 patients, they see a difference in sub-strain dynamics that they say foreshadows a slowing down of the intrinsic rates of strain variation in the sick patients. This slowing could eventually lead to a dysbiosis in the microbial strain community that may portend a shift in the dominant strains of the gut microbiome.<\/p>\n<p>Both of the Bacteroides species are found in high abundance in the gut flora, and they may be keystone species, organisms that help define an entire ecosystem.<\/p>\n<p>Koo and Morrow&#8217;s study, &#8220;Early indicators of microbial strain dysbiosis in the human gastrointestinal microbial community of certain healthy humans and hospitalized COVID?19 patients,&#8221; is published in the journal <em>Scientific Reports.<\/em><\/p>\n<p>Koo and Morrow first analyzed previously published metagenomics data from 41 individuals sampled one year apart and 11 individuals sampled 90 days apart. They looked at a single dominant strain of B. vulgatus in each individual at the two time points to see if they had showed different KEGG metabolic sub-strain patterns, as detected from analysis of single-nucleotide variants in KEGG metabolic pathways, or PKS. In general, most showed a different sub-strain PKS pattern between the two time points of each individual.<\/p>\n<p>  \t\t\t\t\t\t\t<!-- BEGIN mobile-middle-rectangle --> \t\t\t\t\t\t\t\t\t\t\t \t \t\t\t\t\t\t\t\t<\/p>\n<p> \t\t\t\t\t\t\t\t   advertisement \t\t\t\t\t\t\t\t<\/p>\n<p> \t \t\t\t\t\t\t\t\t<!-- \/1002721\/ScienceDaily_Mobile_Middle_Rectangle --> \t\t\t\t\t\t\t\t<\/p>\n<p>\t\t\t\t \t \t\t\t\t\t\t\t\t \t \t\t\t\t\t\t\t \t\t\t\t\t\t\t<!-- END mobile-middle-rectangle --> <\/p>\n<p>The UAB researchers then analyzed previously published metagenomics data from six healthy individuals sampled every few days over three to 10 weeks, again analyzing sub-strains by single-nucleotide variants in 23 KEGG metabolic pathways. Three individuals showed a different sub-strain at every time point, while three showed sub-strains had PKS patterns that appeared, disappeared and reappeared at different time points.<\/p>\n<p>Shared PKS patterns were also seen in two of three hospitalized COVID-19 patients who were sampled multiple times.<\/p>\n<p>&#8220;We suggest that gut microbial communities under stress, such as those found in COVID-19 hospitalized patients, might be in a state indicating the potential shift in which the dominant strain would be outcompeted by a minor strain,&#8221; Koo said. &#8220;Disruptions of the gut microbial community resulting from a strain variation might, in turn, alter the community structure and impact the functions in metabolism and colonization resistance.&#8221;<\/p>\n<p>&#8220;One of the features of a complex biological system is that, as it approaches a critical transition, there is a slowing down of the intrinsic rates of change,&#8221; Morrow said. &#8220;The system enters a condition that is related to autocorrelation, where the patterns would be repeated between time points. It is possible that the shared KEGG metabolic pathway clusters represent a state of autocorrelation in the gut microbial strain community that portends a strain change.&#8221;<\/p>\n<p>Support came from the Marnix E. Heersink School of Medicine at UAB.<\/p>\n<p>Morrow is a professor emeritus of the UAB Department of Cell, Developmental and Integrative Biology, and Koo is a bioinformatician in the UAB Department of Genetics.<\/p>\n<p>The KEGG database is an acronym for the Kyoto Encyclopedia of Genes and Genomes.<\/p>\n<p>Story Source:<\/p>\n<p>Materials provided by University of Alabama at Birmingham. Original written by Jeff Hansen. <em>Note: Content may be edited for style and length.<\/em><\/p>\n<p> <a href=\"https:\/\/www.sciencedaily.com\/releases\/2022\/04\/220427154104.htm\" target=\"_blank\" rel=\"noopener\">Source<\/a><\/p>\n","protected":false},"excerpt":{"rendered":"<p>Casey D. Morrow, Ph.D., and colleagues at the University of Alabama at Birmingham previously developed a microbiome &#8220;fingerprint&#8221; method called WSS that identifies single strains of particular gut bacteria, through analysis of&#8230;<\/p>\n","protected":false},"author":7,"featured_media":2262,"comment_status":"open","ping_status":"open","sticky":false,"template":"","format":"standard","meta":[],"categories":[9],"tags":[],"_links":{"self":[{"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/posts\/2261"}],"collection":[{"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/posts"}],"about":[{"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/types\/post"}],"author":[{"embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/users\/7"}],"replies":[{"embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/comments?post=2261"}],"version-history":[{"count":0,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/posts\/2261\/revisions"}],"wp:featuredmedia":[{"embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/"}],"wp:attachment":[{"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/media?parent=2261"}],"wp:term":[{"taxonomy":"category","embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/categories?post=2261"},{"taxonomy":"post_tag","embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/tags?post=2261"}],"curies":[{"name":"wp","href":"https:\/\/api.w.org\/{rel}","templated":true}]}}