{"id":1356,"date":"2022-04-11T07:02:26","date_gmt":"2022-04-11T07:02:26","guid":{"rendered":"https:\/\/alynews.org\/new-method-for-probing-the-bewildering-diversity-of-the-microbiome\/"},"modified":"2022-04-11T07:02:26","modified_gmt":"2022-04-11T07:02:26","slug":"new-method-for-probing-the-bewildering-diversity-of-the-microbiome","status":"publish","type":"post","link":"http:\/\/localhost\/m\/new-method-for-probing-the-bewildering-diversity-of-the-microbiome\/","title":{"rendered":"New method for probing the bewildering diversity of the microbiome"},"content":{"rendered":"<p><img src=\"https:\/\/www.sciencedaily.com\/images\/scidaily-icon.png\" alt=\"New method for probing the bewildering diversity of the microbiome\"><\/p>\n<p>In recent years, researchers have begun to explore the vast assemblage of microbes on and within the human body. These include protists, archaea, fungi, viruses and vast numbers of bacteria living in symbiotic ecosystems.<\/p>\n<p>Known collectively as the human microbiome, these tiny entities influence an astonishing range of activities, from metabolism to behavior and play a central role in health and disease. Some 39 trillion non-human microbes flourish on and within us, in a ceaseless, interdependent bustle. Together, they make up over half of the human body&#8217;s cells, though they may possess 500 times as many genes as are found in human cells. Identifying and making sense of this microbial m\u00e9lange has been a central challenge for researchers.<\/p>\n<p>In a new study, Qiyun Zhu and his colleagues describe a new method for probing the microbiome in unprecedented detail. The technique provides greater simplicity and ease of use compared with existing approaches. Using the new technique, the researchers demonstrate an improved ability to pinpoint biologically relevant characteristics, including a subject&#8217;s age and sex based on microbiome samples.<\/p>\n<p>The innovative research holds the promise of rapidly advancing investigations into the mysteries of the microbiome. With such knowledge, researchers hope to better understand how these microbes collectively act to safeguard human health and how their dysfunction can lead to a broad range of diseases. In time, drugs and other therapies may even be tailor-made based on a patient&#8217;s microbiomic profile.<\/p>\n<p>Professor Zhu is a researcher in the Biodesign Center for Fundamental and Applied Microbiology and ASU&#8217;s School of Life Sciences. The research team includes collaborators from the University of California, San Diego, including co-corresponding author Rob Knight, Zhu&#8217;s former mentor.<\/p>\n<p>The group&#8217;s research results appear in the current issue of the journal <em>mSystems<\/em>.<\/p>\n<p>  \t\t\t\t\t\t\t<!-- BEGIN mobile-middle-rectangle --> \t\t\t\t\t\t\t\t\t\t\t \t \t\t\t\t\t\t\t\t<\/p>\n<p> \t\t\t\t\t\t\t\t   advertisement \t\t\t\t\t\t\t\t<\/p>\n<p> \t \t\t\t\t\t\t\t\t<!-- \/1002721\/ScienceDaily_Mobile_Middle_Rectangle --> \t\t\t\t\t\t\t\t<\/p>\n<p>\t\t\t\t \t \t\t\t\t\t\t\t\t \t \t\t\t\t\t\t\t \t\t\t\t\t\t\t<!-- END mobile-middle-rectangle --> <\/p>\n<p>Tools of the trade<\/p>\n<p>Two powerful technologies have been used to help researchers unlock the diversity and complexity of the microbiome, by sequencing the microbial DNA present in a sample. These are known as 16S and metagenomic sequencing. The technique described in the current study draws on the strengths of both methods to create a new way of processing data from the microbiome.<\/p>\n<p>&#8220;We borrow some of the wisdom that developed from 16S RNA sequencing and apply it to metagenomics,&#8221; Zhu says. Unlike other sequencing methods, including 16S, metagenomics allows researchers to sequence all the DNA information present in a microbiome sample. But the new study shows that the metagenomic approach has room for improvement. &#8220;The way people currently analyze metagenomic data is limited, because whole genome data has to first be translated into taxonomy.&#8221;<\/p>\n<p>The new technique, known as Operational Genomic Units (OGU) does away with the laborious and sometimes misleading practice of assigning taxonomic categories like genus and species to the multitude of microbes present in a sample. Instead, the method uses individual genomes as the basic units for statistical analysis and simply attempts to align sequences present in a sample to sequences found in existing genomic databases.<\/p>\n<p>By doing this, researchers can get much more fine-grained resolution, which is particularly useful when microbes are present that are closely related in DNA sequence. This is true because most taxonomic classifications are based on sequence similarity. If two sequences differ by less than a certain threshold, they fall into the same taxonomic category, however the OGU approach can help researchers tell them apart.<\/p>\n<p>  \t\t\t\t\t\t<!-- BEGIN mobile-bottom-rectangle --> \t\t\t\t\t\t\t\t\t\t \t \t\t\t\t\t\t\t<\/p>\n<p> \t\t\t\t\t\t\t   advertisement \t\t\t\t\t\t\t<\/p>\n<p> \t \t\t\t\t\t\t\t<!-- \/1002721\/ScienceDaily_Mobile_Bottom_Rectangle --> \t\t\t\t\t\t\t<\/p>\n<p>\t\t\t\t \t \t\t\t\t\t\t\t \t \t\t\t\t\t\t \t\t\t\t\t\t<!-- END mobile-bottom-rectangle --> <\/p>\n<p>Further, the method overcomes errors in taxonomy that persist as relics from the pre-sequencing epoch, when different species were defined by their morphology rather than from DNA sequence data.<\/p>\n<p>In addition to improvements in resolution and simplicity, OGU can help researchers analyze data using what are known as phylogenetic trees. As the name implies, these are branching structures that can describe the degree of relatedness between organisms, based on their sequence similarity. Just as two distantly related species like worms and antelope will appear on more distant branches of a phylogenetic tree, so will more distantly related bacteria and other constituents of the microbiome.<\/p>\n<p>Innovations in sequencing<\/p>\n<p>The most widely used technique for probing the microbiome, known as 16S ribosomal RNA sequencing or just 16S, relies on a simple idea. All bacteria have a 16S gene, which is essential to the machinery bacteria need to initiate protein synthesis. The bacterial 16S gene, measuring 1500 base pairs in length, consists of distinct regions. Some of these regions change very little between different bacteria and over evolutionary timeframes, while others are highly variable.<\/p>\n<p>Researchers realized that the conserved and variable regions of the 16S gene allow it to act as a molecular clock, keeping track of bacteria that are more closely or more distantly related, based on their sequence similarity. Thus, the 8 conserved and 9 variable regions of 16S can be used to fingerprint bacteria.<\/p>\n<p>To do this, a microbiome sample is first collected. This could be a fecal sample, to evaluate the gut microbiome, or a sample from the skin or from the mouth. Each body site is home to a different bacterial menagerie.<\/p>\n<p>Next, PCR technology is used to amplify portions of the 16S gene. By sequencing highly conserved regions, a broad swath of bacteria can be identified, while sequencing of variable regions helps narrow the identity of particular bacteria.<\/p>\n<p>Although 16S is an inexpensive and well-developed method, it has limitations. The technique can only give a general idea of the kinds of bacteria present, with limited resolution. In general, 16S is only accurate to the genus level of identification.<\/p>\n<p>Enter metagenomic sequencing. This technique sequences the full genomes of all microbes present in a microbiome sample, (not just bacteria, as with 16S). Metagenomics allows researchers to sequence thousands of organisms in parallel, providing accurate, species-level resolution. The greater resolution however does come with costs. Metagenomic data is far richer and more computationally challenging to analyze than 16S data and more expensive in time and money to process.<\/p>\n<p>A new path for metagenomics<\/p>\n<p>The OGU technique streamlines metagenomic sequencing, while providing even greater resolution. The approach classifies microbes in a sample strictly according to their alignment with a reference database &#8212; no taxonomic assignment required. The approach enables researchers to evaluate the degree of species diversity present in a sample.<\/p>\n<p>Compared with 16S and standard metagenomic sequencing, the new approach is superior in ferreting out biologically relevant information. Using the classic Human Microbiome Project dataset of 210 metagenomes sampled from seven body sites of male and female human subjects, the study demonstrates better correlation between body site and host sex.<\/p>\n<p>Next, 6,430 stool samples collected through a random sampling of the Finnish population were analyzed, using both 16S and metagenomic sequencing. The samples belong to a large, randomly sampled cohort of the Finnish population, known as FINRISK. The aim was to predict the age of sampled individuals, based on gut microbial composition. Again, the OGU method outperformed 16S and conventional metagenomic analysis, providing more accurate predictions.<\/p>\n<p>New research drawing on still larger datasets will further enhance the resolution of the new technique and expand the descriptive power of taxonomy-independent analysis.<\/p>\n<p> <a href=\"https:\/\/www.sciencedaily.com\/releases\/2022\/04\/220404185543.htm\" target=\"_blank\" rel=\"noopener\">Source<\/a><\/p>\n","protected":false},"excerpt":{"rendered":"<p>In recent years, researchers have begun to explore the vast assemblage of microbes on and within the human body. These include protists, archaea, fungi, viruses and vast numbers of bacteria living in&#8230;<\/p>\n","protected":false},"author":7,"featured_media":1357,"comment_status":"open","ping_status":"open","sticky":false,"template":"","format":"standard","meta":[],"categories":[9],"tags":[],"_links":{"self":[{"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/posts\/1356"}],"collection":[{"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/posts"}],"about":[{"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/types\/post"}],"author":[{"embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/users\/7"}],"replies":[{"embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/comments?post=1356"}],"version-history":[{"count":0,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/posts\/1356\/revisions"}],"wp:featuredmedia":[{"embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/"}],"wp:attachment":[{"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/media?parent=1356"}],"wp:term":[{"taxonomy":"category","embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/categories?post=1356"},{"taxonomy":"post_tag","embeddable":true,"href":"http:\/\/localhost\/m\/wp-json\/wp\/v2\/tags?post=1356"}],"curies":[{"name":"wp","href":"https:\/\/api.w.org\/{rel}","templated":true}]}}